public/login.html

```diff
@@ -30,21 +31,35 @@ <h2 class="mdl-card__title-text">Sign In</h2>
                     <label class="mdl-textfield__label" for="uid">username</label>
                 </div>
             </form>
+            <p>This site is for dev testing only.<br><span style="color: red; font-weight: bold;">Do not submit any
+                    personal data.</span></p>
         </div>
         <div class="mdl-card__actions mdl-card--border">
             <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" id="signInButton">
                 Sign In
             </a>
         </div>
     </div>
+    <div id="loginFooter">
+        <p class="text-italic">aka.ms/webauthntest</p>
+        <p id="version" class="text-muted"></p>
+    </div>
```

Comment:
Can you also add this link to README file?

Reply:
done